2020-07-22 09:14:17.015 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:14:27.079 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:14:37.172 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:14:47.271 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:14:57.370 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:15:07.470 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:15:17.571 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:15:27.770 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:15:38.070 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:15:48.673 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:17:38.840 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: Reconnection attempt exceeded timeout of 60 SECONDS 
2020-07-22 09:17:41.632 [HikariPool-1 housekeeper] WARN  com.zaxxer.hikari.pool.HikariPool - HikariPool-1 - Thread starvation or clock leap detected (housekeeper delta=2m15s390ms643Âµs720ns).
2020-07-22 09:17:50.965 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:18:05.164 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:18:23.365 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:18:49.780 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:19:29.880 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:20:09.969 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:20:50.067 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:21:30.166 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:22:10.266 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:22:50.365 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:23:30.467 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:24:10.568 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:24:50.672 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 09:25:30.781 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-22 11:33:24.388 [http-nio-8085-exec-1] WARN  com.zaxxer.hikari.util.DriverDataSource - Registered driver with driverClassName=com.mysql.jdbc.Driver was not found, trying direct instantiation.
2020-07-22 15:10:21.715 [http-nio-8085-exec-1] WARN  com.zaxxer.hikari.util.DriverDataSource - Registered driver with driverClassName=com.mysql.jdbc.Driver was not found, trying direct instantiation.
